packages/artillery-plugin-publish-metrics/lib/open-telemetry/vendor-translators.js

```diff
@@ -64,11 +64,14 @@ const vendorTranslators = {
     return otelTemplate(config, dynatraceTraceSettings);
   },
   'open-telemetry': (config) => {
-    let tracesConfig = config;
+    let newConfig = config;
```

Comment:
Changing name to `newConfig` as makes more sense, especially now when adding the `metrics` check as well.